api/src/main/java/org/openmrs/PersonName.java

```diff
@@ -372,6 +374,7 @@ public Boolean getPreferred() {
 	/**
 	 * @param preferred The preferred to set.
 	 */
+	@Override
 	public void setPreferred(Boolean preferred) {
```

Comment:
i would try looking through this method and see whether there is duplication of it

Reply:
Some small changes .Travis failure is due to PersonPatientSharable.java, also i would advise to first run your changes before you push otherwise keep up

Reply:
I have resolved the Travis build failures

Reply:
Thanks for this again, can you first squash your commits first